src/popup/modules/QrLib/qrerr.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Errors in QR code generation
+ *
+ * @module QrLib/qrerr
```

Comment:
Please just use the long form for the file name. `QrErrors` is not so long one needs to abbreviate it. 😉 

That said, I would have been perfectly fine with including this inside of `QrCreator` or so, but this way it is fine too. 😃